Assignment-2-Crud(updated).c

```diff
@@ -0,0 +1,231 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct User
+{
+    int Id;
+    char Name[50];
+    int Age;
+} User;
+
+void addUser();
+void displayUsers();
+void updateUser();
+void deleteUser();
+int checkUserExists(int id); 
+User getInputUserDetails();  
+void printUserDetails(User user); 
+
+int main()

```

Comment:
no need of separate new file, you should edit the same file for fixing review comments and need to push the code, please refer any online source like how to fix the review comments and push the code.